doc/evm_incompatibilities.md

```diff
@@ -8,7 +8,7 @@ Following EIPs are not supported by our EVM:
 
 
 ## Latest supported solc version
-We currently support only `0.8.24` as [EIP-5656](https://eips.ethereum.org/EIPS/eip-5656) is not yet supported.
+We currently support latest
```

Comment:
Maybe it's better to write exact version as latest could be changed and this will out of date